### DISCUSSION TITLE
Arrow keys input not working when using a different input language than English

### DISCUSSION THREAD

**Discussion by TeodorDyakov** [hajimehoshi/ebiten#1552] [Q&A]
When I start the game using a different language than English(in my case Bulgarian) the arrow keys do not work. 
THe problem appears on Ubuntu OS 20.04.

**Comment by hajimehoshi** [comment]
Could you show the code to explain the issue? Thanks.

**Comment by TeodorDyakov** [comment]
Of course The problem appears when I use ebiten.IsKeyPressed fucntion. If i run the program while the keyboard input is set to Bulgarian the key presses will not be registered. However If I start the game with English Input language it works fine.

Here is the code.
```
if ebiten.IsKeyPressed(ebiten.KeyLeft) {
	ent.x -= dx
}
```

**Reply by hajimehoshi** [reply]
Ebiten's key API assumes US keyboard and the key constant names are always US's. So the key name might not match with other languages' keyboards. `:` in Japanese keyboard is represented by `'` (`ebiten.KeyApostrophe`) for example.
